core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -66,19 +68,64 @@
 
   private ConfigurationUtils() {} // prevent instantiation
 
+  /**
+   * Gets the embedded journal addresses to use for the given service type (either master-raft or
+   * job-master-raft).
+   *
+   * @param conf configuration
+   * @param serviceType the service to get addresses for
+   * @return the addresses
+   */
+  public static List<InetSocketAddress> getEmbeddedJournalAddresses(AlluxioConfiguration conf,
+      ServiceType serviceType) {
+    Preconditions.checkState(
+        serviceType == ServiceType.MASTER_RAFT || serviceType == ServiceType.JOB_MASTER_RAFT);
+    PropertyKey addressKey = PropertyKey.MASTER_EMBEDDED_JOURNAL_ADDRESSES;
+    if (serviceType == ServiceType.JOB_MASTER_RAFT
+        && conf.isSet(PropertyKey.JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES)) {
+      addressKey = PropertyKey.JOB_MASTER_EMBEDDED_JOURNAL_ADDRESSES;
+    }
+    List<InetSocketAddress> inetAddresses = new ArrayList<>();
+    if (conf.isSet(addressKey)) {
```

Comment:
Could we do 
```
if (!conf.isSet(addressKey)) {
  return Arrays.asList(NetworkAddressUtils.getConnectAddress(serviceType, conf));
}
...
```

Reply:
switched the conditions to use early return instead of else